<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>P_MouseResizeButtonPart</class>
 <widget class="QWidget" name="P_MouseResizeButtonPart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>150</width>
    <height>29</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_group" native="true">
     <property name="styleSheet">
      <string notr="true">
QToolButton{
	padding:2px;
	border: 1px solid transparent;
	background: rgba(255,255,255,0);
}
QToolButton:hover{
	border-radius:1px;
	border:1px solid #f2ca58;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #fdecad, stop:0.3 #fde48c, stop:0.9 #fce58c, stop:1 #fdfadf )
}
QToolButton:pressed{
	border-radius:1px;
	border:1px solid #f2ca58;
	background-color: #FDF4BF;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="toolButton_zoom_in">
        <property name="toolTip">
         <string>放大</string>
        </property>
        <property name="text">
         <string>放大</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../../../../DrillGIFActionSequenceEditor/trunk/DrillGIFActionSequenceEditor/drillgifactionsequenceeditor.qrc">
          <normaloff>:/DrillGIFActionSequenceEditor/Resources/icons/view/Zoom_In.png</normaloff>:/DrillGIFActionSequenceEditor/Resources/icons/view/Zoom_In.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_zoom_out">
        <property name="toolTip">
         <string>缩小</string>
        </property>
        <property name="text">
         <string>缩小</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../../../../DrillGIFActionSequenceEditor/trunk/DrillGIFActionSequenceEditor/drillgifactionsequenceeditor.qrc">
          <normaloff>:/DrillGIFActionSequenceEditor/Resources/icons/view/Zoom_Out.png</normaloff>:/DrillGIFActionSequenceEditor/Resources/icons/view/Zoom_Out.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_zoom_regular">
        <property name="toolTip">
         <string>原比例</string>
        </property>
        <property name="text">
         <string>原比例</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../../../../DrillGIFActionSequenceEditor/trunk/DrillGIFActionSequenceEditor/drillgifactionsequenceeditor.qrc">
          <normaloff>:/DrillGIFActionSequenceEditor/Resources/icons/view/Zoom_Regular.png</normaloff>:/DrillGIFActionSequenceEditor/Resources/icons/view/Zoom_Regular.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_zoomValue">
        <property name="text">
         <string>100.0%</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../../../../DrillGIFActionSequenceEditor/trunk/DrillGIFActionSequenceEditor/drillgifactionsequenceeditor.qrc"/>
 </resources>
 <connections/>
</ui>
